frontend/app/assets/stylesheets/spree/frontend/views/spree/shared/nav_bar.scss

```diff
@@ -42,6 +42,7 @@
 
       form {
         input#keywords{
+          background: transparent;
           color: theme-color("dark-text");
```

Comment:
@pmilewski @AleksandraKaminska shouldn't this use `$meganav-font-color` ?

Reply:
Yes, I think it should